app/products/calls/connection/websocket_event_handlers.ts

```diff
@@ -233,7 +233,7 @@ export const handleHostLowerHand = async (serverUrl: string, msg: WebSocketMessa
 export const handleHostRemoved = async (serverUrl: string, msg: WebSocketMessage<HostControlsMsgData>) => {
     const currentCall = getCurrentCall();
     if (currentCall?.serverUrl !== serverUrl ||
-        currentCall?.channelId !== msg.data.channel_id ||
+        currentCall?.channelId !== msg.broadcast.channel_id ||
```

Comment:
Is this the payload in https://github.com/mattermost/mattermost-plugin-calls/blob/8f36a15d23c2086aa3489e3e21e5f2bbbee6118a/server/host_controls.go#L196-L200? If so, channel ID should be there as expected.

Reply:
@streamer45 It's this PR that changed it: https://github.com/mattermost/mattermost-plugin-calls/pull/731

Reply:
@cpoile In general I'd try to keep all the info we need in-payload because we have some use cases where the broadcast may not be useful, for example:

- Call-specific broadcasts (coming in https://github.com/mattermost/mattermost-plugin-calls/pull/609)
- Our bot user can't rely on broadcast since it's not required (or possible, e.g. DMs) to be in the channel.

If none applies here it's fine, I mostly wanted to raise it for awareness.

Reply:
@streamer45 Hmm, that could be a breaking change then because I think in some cases we rely on the broadcast data elsewhere. Good to know. I'll continue sending it then here.

Reply:
I'll need to review, I think you can actually send a broadcast with both channel id and user id set but the latter would take precedence so we should be good in terms of keeping compatibility but a general review would be best just to avoid problems in the future. It's better to trust what we control, in general (CSS included) :p